web/modules/custom/nerdcustom/nerdcustom.module

```diff
@@ -102,3 +103,21 @@ function nerdcustom_preprocess_node(&$variables) {
     nerdcustom_canonical_helper($variables, $variables['node']);
   }
 }
+
+/**
+ * Implements hook_theme().
+ */
+function nerdcustom_theme() {
+  return array(
```

Comment:
@stevector, If you're providing a custom theme implementation in this module, there should be a twig file in the module too. One of the twig files add to the custom theme should be moved here.